packages/gatsby-cli/src/create-cli.js

```diff
@@ -83,6 +83,10 @@ function buildLocalCommands(cli, isLocalSite) {
   function getCommandHandler(command, handler) {
     return argv => {
       report.setVerbose(!!argv.verbose)
+      if (argv.noColor) {
+        process.env.FORCE_COLOR = `0`
```

Comment:
Small comment--but we usually add comments here, since this a Chalk documented variable right?

e.g.

```suggestion
        // disables colors in chalk
        // see: https://www.npmjs.com/package/chalk#chalksupportscolor
        process.env.FORCE_COLOR = `0`
```

Reply:
This disables colors in `chalk@2` and in `ansi-colors` - it's problematic in `chalk@1` (which is why I needed to fork `friendly-errors-webpack-plugin` package)